data/BGC0000002.json

```diff
@@ -18,6 +18,11 @@
                 "AAAAAAAAAAAAAAAAAAAAAAAA"
             ],
             "version": "2.0"
+        },
+        {
+            "comments": [],
+            "contributors": [],
+            "version": "3.0"
```

Comment:
All these empty changelogs should be removed.

Reply:
Currently, substrate names are only capitalised when they are proteinogenic (mostly due to the check for proteinogenic AA names in the current json schema). I'd suggest to make everything lower-case for consistency, as auto-capitalisation of other substrates is not trivial.

Will rebase and remove empty changelogs.

Reply:
I agree, we should change the schema to not capitalise the amino acids. That's just been transferred in from the old schema, I guess.